modules/functions-list-kn.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies
+
+// * /serverless/cli_reference/kn-func-ref.adoc
+
+[id="functions-list-kn_{context}"]
+= Listing existing functions
+
+You can list existing functions by using `kn func list`. If you want to list functions that have been deployed as Knative services, you can also use `kn service list`.
+
+[NOTE]
+====
+If the namespace is not specified for the following commands, this defaults to the current namespace.
```

Comment:
This is fine the way it is, but I was just curious why it was necessary here. Isn't kn kind of like oc, where if you don't specify the namespace, then it defaults to the current namespace?

Reply:
Yeah actually maybe it's not necessary tbh, I'll remove it 🙂